openapi.yaml

```diff
@@ -8578,3 +8626,142 @@ components:
+        date_from:
+          description: Date from when to search for the deployments,
+          format: date
+          type: string
+        date_to:
+          description: Date up to which to search for the deployments.
+          format: date
+          type: string
+        timezone:
+          $ref: '#/components/schemas/TimeZone'
+        account:
+          description: Session account ID.
+          type: integer
+        in:
+          $ref: '#/components/schemas/RepositorySet'
+        with:
+          $ref: '#/components/schemas/ReleaseWith'
+        pr_labels_include:
+          description: At least one of these labels must be present in at least one
+            deployed PR.
+          items:
+            type: string
+          type: array
+        pr_labels_exclude:
+          description: None of these labels must be present in each deployed PR.
+          items:
+            type: string
+          type: array
+        jira:
+          $ref: '#/components/schemas/JIRAFilter'
+        environments:
+          description: Deployments must belong to one of these environments.
+          items:
+            type: string
+          type: array
+        conclusions:
+          description: Deployments must execute with any of these conclusions.
+          items:
+            $ref: '#/components/schemas/DeploymentConclusion'
+          type: array
+        with_labels:
+          description: Deployments should contain at least one of the specified label
+            values. `null` matches any label value and effectively checks the label
+            existence.
+          type: object
+        without_labels:
+          description: Deployments may not contain the specified label values. `null`
+            matches any label value and effectively ensures that the label does not
+            exist.
+          type: object
+      required:
+      - account
+      - date_from
+      - date_to
+      type: object
+    FilteredDeployments:
+      description: Found deployments, response from `/filter/deployments`.
+      items:
+        $ref: '#/components/schemas/FilteredDeployment'
+      type: array
+    DeploymentAnalysis:
+      additionalProperties: false
+      description: Result the analysis of the deployment.
+      properties:
+        code:
+          additionalProperties: false
+          description: Summary of the deployed code.
+          properties:
+            prs:
+              description: Number of deployed pull requests per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+            lines_prs:
+              description: Number of changed lines in the deployed PRs per repository.
+              additionalProperties:
+                minimum: 0
+                type: integer
+              type: object
+            lines_overall:
+              description: Number of changed lines in the deployment per repository.
```

Comment:
What is the difference between `lines_prs` and `lines_overall`? the changed lines of the commits pushed outside PRs?

Reply:
`lines_overall` is the full number of lines, both in commits in PRs and in commits bypassing PRs. `lines_prs` is the number of lines of commits in pull requests only.

Reply:
I added this distinction because I know it will be a very cheap effort to support it.